modules/dnn/src/layers/softmax_layer.cpp

```diff
@@ -76,7 +76,7 @@ class SoftMaxLayerImpl CV_FINAL : public SoftmaxLayer
 
     SoftMaxLayerImpl(const LayerParams& params)
     {
-        axisRaw = params.get<int>("axis", 1);
+        axisRaw = params.get<int>("axis", -1);
```

Comment:
`axisRaw` should be normalize in `finalize` to avoid negative value.

Reply:
`axisRaw` was defined here before, and I can move it to `finalize` if necessary

Reply:
No, I mean `axisRaw` get its value here but normalize in `finalize()`.

However, if the softmax kernel `softmax()` and `logSoftmax()` supports negative axis, it is fine.

Reply:
Yes, kernel will normalize axis again.

Reply:
Okay, thank you.